### PR TITLE
CPB-12: Static and media configurations

### PR DESCRIPTION
**Pull request by LV-coding**

### CODE REVIEW

**Review by pazuzu-ua — COMMENTED** [review]
All is good, but please ensure PEP8 and use pre-commit checks. You need to add empty line in urls.py and change **'** to **"**.

**Comment by LV-coding** [comment]
> All is good, but please ensure PEP8 and use pre-commit checks. You need to add empty line in urls.py and change **'** to **"**.

I added empty line in other commit. What do you meen in 'change "to"'?

**Comment by pazuzu-ua** [comment]
> > All is good, but please ensure PEP8 and use pre-commit checks. You need to add empty line in urls.py and change **'** to **"**.
> 
> I added empty line in other commit. What do you meen in 'change "to"'?

You have single quotation marks to double

**Review by pazuzu-ua — APPROVED** [review]

File: backend/config/settings/base.py [hunk 1]
```diff
@@ -120,6 +122,29 @@
 # https://docs.djangoproject.com/en/4.1/howto/static-files/
 
 STATIC_URL = "static/"
+STATIC_ROOT = os.path.join(BASE_DIR, "staticfiles")
+
```
Comment (devoorhees) [inline backend/config/settings/base.py]: Add compression and caching support for staticfiles:
`STATICFILES_STORAGE = "whitenoise.storage.CompressedManifestStaticFilesStorage"`

File: backend/config/settings/base.py [hunk 2]
```diff
@@ -22,7 +22,7 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
 
-ALLOWED_HOSTS = []
+ALLOWED_HOSTS = ["*"]
```
Comment (devoorhees) [inline backend/config/settings/base.py]: Move this to settings/dev.py and leave `ALLOWED_HOSTS` empty in the base.py.

File: backend/config/settings/test.py [hunk 3]
```diff
@@ -1 +0,0 @@
-from .base import *
```
Comment (devoorhees) [inline backend/config/settings/test.py:1]: Revert changes and add `# noqa` at the end of this line to exclude formatting.

**Review by devoorhees — APPROVED** [review]

**Pull request merged**